tests/fixtures/hybrid/hybrid_fixtures.py

```diff
@@ -0,0 +1,275 @@
+        geometry=geometry,
+        liquid=fuel_fluid,
+        gas=fuel_pressurant,
+        ullage=ullage,
+    )
+
+    return fuel_tank
+
+
+@pytest.fixture
+def oxidizer_tank(oxidizer_fluid, oxidizer_pressurant):
+    """An example of a oxidizer cylindrical tank with spherical
+    caps.
+
+    Parameters
+    ----------
+    oxidizer_fluid : rocketpy.Fluid
+        Oxidizer fluid of the tank. This is a pytest fixture.
+    oxidizer_pressurant : rocketpy.Fluid
+        Pressurizing fluid of the oxidizer tank. This is a pytest
+        fixture.
+
+    Returns
+    -------
+    rocketpy.UllageBasedTank
+    """
+    geometry = CylindricalTank(0.0744, 0.8068, spherical_caps=True)
+    ullage = (
+        -Function("data/SEBLM/test124_Lox_Volume.csv") * 1e-3 + geometry.total_volume
+    )
+    oxidizer_tank = UllageBasedTank(
+        name="Lox Tank",
+        flux_time=(8, 20),
+        geometry=geometry,
+        liquid=oxidizer_fluid,
+        gas=oxidizer_pressurant,
+        ullage=ullage,
+    )
+
+    return oxidizer_tank
+
+
+@pytest.fixture
+def liquid_motor(pressurant_tank, fuel_tank, oxidizer_tank):
+    """An example of a liquid motor with pressurant, fuel and oxidizer tanks.
+
+    Parameters
+    ----------
+    pressurant_tank : rocketpy.MassBasedTank
+        Tank that contains pressurizing fluid. This is a pytest fixture.
+    fuel_tank : rocketpy.UllageBasedTank
+        Tank that contains the motor fuel. This is a pytest fixture.
+    oxidizer_tank : rocketpy.UllageBasedTank
+        Tank that contains the motor oxidizer. This is a pytest fixture.
+
+    Returns
+    -------
+    rocketpy.LiquidMotor
+    """
+    liquid_motor = LiquidMotor(
+        thrust_source="data/SEBLM/test124_Thrust_Curve.csv",
+        burn_time=(8, 20),
+        dry_mass=10,
+        dry_inertia=(5, 5, 0.2),
+        center_of_dry_mass_position=0,
+        nozzle_position=-1.364,
+        nozzle_radius=0.069 / 2,
+    )
+    liquid_motor.add_tank(pressurant_tank, position=2.007)
+    liquid_motor.add_tank(fuel_tank, position=-1.048)
+    liquid_motor.add_tank(oxidizer_tank, position=0.711)
+
+    return liquid_motor
+
+
+@pytest.fixture
+def spherical_oxidizer_tank(oxidizer_fluid, oxidizer_pressurant):
+    """An example of a oxidizer spherical tank.
+
+    Parameters
+    ----------
+    oxidizer_fluid : rocketpy.Fluid
+        Oxidizer fluid of the tank. This is a pytest fixture.
+    oxidizer_pressurant : rocketpy.Fluid
+        Pressurizing fluid of the oxidizer tank. This is a pytest
+        fixture.
+
+    Returns
+    -------
+    rocketpy.UllageBasedTank
+    """
+    geometry = SphericalTank(0.05)
+    liquid_level = Function(lambda t: 0.1 * np.exp(-t / 2) - 0.05)
+    oxidizer_tank = LevelBasedTank(
+        name="Lox Tank",
+        flux_time=10,
+        geometry=geometry,
+        liquid=oxidizer_fluid,
+        gas=oxidizer_pressurant,
+        liquid_height=liquid_level,
+    )
+
+    return oxidizer_tank
+
```

Comment:
I think this is in the wrong place, it is duplicated from the liquid_motor_fixtures.py file.  Am I right? Can you please delete these lines if yes, please?